<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="butAuto">
        <property name="text">
         <string>Ручная</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxSetArgs">
        <property name="title">
         <string>Параметры</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="butSet">
           <property name="text">
            <string>Установить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labArg1">
           <property name="text">
            <string>a</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditArg1"/>
         </item>
         <item>
          <widget class="QLabel" name="labArg2">
           <property name="text">
            <string>b</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditArg2"/>
         </item>
         <item>
          <widget class="QLabel" name="labArg3">
           <property name="text">
            <string>c</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditArg3"/>
         </item>
         <item>
          <widget class="QLabel" name="labArg4">
           <property name="text">
            <string>d</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditArg4"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="GlWidget" name="widget" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="butLine">
        <property name="text">
         <string>Кривая</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="butCrear">
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>438</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuReg">
    <property name="title">
     <string>Регрессия</string>
    </property>
    <addaction name="actionLin"/>
    <addaction name="actionSin"/>
   </widget>
   <addaction name="menuReg"/>
  </widget>
  <action name="actionLin">
   <property name="text">
    <string>Линейная</string>
   </property>
  </action>
  <action name="actionSin">
   <property name="text">
    <string>Синусоида</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GlWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
